src/stories/tabs/Tabs.module.scss

```diff
@@ -12,20 +11,72 @@
     &::-webkit-scrollbar {
         display: none;
     }
+
+    &.thick {
+        padding-bottom: 4px;
+    }
+    &.thin {
+        padding-bottom: 1px;
+    }
+
+    &.full {
+        justify-content: stretch;
+    }
+}
+
+.tabBackground {
+    &.glass {
+        position: absolute;
+        width: 100%;
+    }
+}
+
+.tabGroup {
+    width: 100%;
+    overflow: auto;
+    height: 100%;
+}
+.glassContainer {
```

Comment:
not sure why our eslint isn't catching this but ideally we should have newlines between rulesets